src/test/java/lotto/domain/LottoPaperTest.java

```diff
@@ -0,0 +1,23 @@
+package lotto.domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+class LottoPaperTest {
+    LottoPaper lottoPaper;
+
+    @BeforeEach
+    public void createLottoPaper() {
+        lottoPaper = new LottoPaper(Arrays.asList(1, 2, 3, 4, 5, 6));
+    }
+
+
+    @Test
+    void getLottoNumbers() {
+        Assertions.assertThat(lottoPaper.getLottoNumbers()).isEqualTo(Arrays.asList(1, 2, 3, 4, 5, 6));
+    }
+
+}
```

Comment:
불필요한 공백은 제거해주시면 좋을 것 같아요!
```suggestion
    @BeforeEach
    public void createLottoPaper() {
        lottoPaper = new LottoPaper(Arrays.asList(1, 2, 3, 4, 5, 6));
    }

    @Test
    void getLottoNumbers() {
        Assertions.assertThat(lottoPaper.getLottoNumbers()).isEqualTo(Arrays.asList(1, 2, 3, 4, 5, 6));
    }
}
```